opentracing-api/src/main/java/io/opentracing/propagation/Format.java

```diff
@@ -71,5 +71,16 @@
          * @see Format
          */
         public final static Format<ByteBuffer> BINARY = new Builtin<ByteBuffer>();
+
+        /**
+         * @return Short name for built-in formats as they tend to show up in exception messages.
+         */
+        @Override
+        public String toString() {
```

Comment:
Agree. I wasn't sure whether the public constructor was intentional.
In this case, I would propose changing Builtin to an enum, which automatically also fixes the toString() issue!